db/schema.rb

```diff
@@ -10,13 +10,29 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170805160229) do
+ActiveRecord::Schema.define(version: 20180220204849) do
 
   create_table "books", force: :cascade do |t|
     t.string "title"
     t.string "isbn"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "publisher"
+  end
+
+  create_table "shop_stocks", force: :cascade do |t|
+    t.integer "book_id"
+    t.integer "shop_id"
+    t.integer "value"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["book_id"], name: "index_shop_stocks_on_book_id"
+    t.index ["shop_id"], name: "index_shop_stocks_on_shop_id"
```

Comment:
I don't see any unique validation on the Stock::Shop model but there should be unique constrain for shop_id and book_id otherwise there might be a case when 2 the same books in a shop and we don't want it :)
`add_references [:shop_id, :book_id], unique: true`